cmd/gardener-extension-runtime-gvisor/app/app.go

```diff
@@ -19,6 +19,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/gardener/gardener-extension-runtime-gvisor/pkg/version"
```

Comment:
This component should not maintain an "own version" of the version flag but should rather reuse the `k8s.io/component-base/version/verflag` pkg. Example - https://github.com/gardener/gardener-extension-provider-alicloud/pull/247.

Reply:
Based on my first stab at it, the verflag package seems to implement a "--version" flag, which is functionally different from the "version" subcommand originally requested. @vpnachev, can you shed some light - is the requirement still for a subcommand?

Reply:
At the end the effect is the same. I would strongly vote to use `k8s.io/component-base/version/verflag` - it is already used by gardener core components, K8s core components and provider-alicloud; it adds additional unified metadata such as go version, arch and others. It does not make sense each and every extension to come up with its own version command that displays different things.